src/client/clientmap.h

```diff
@@ -37,25 +37,6 @@ struct MapDrawControl
 	bool show_wireframe = false;
 };
 
-struct MeshBufList
-{
-	video::SMaterial m;
-	std::vector<std::pair<v3s16,scene::IMeshBuffer*>> bufs;
-};
-
-struct MeshBufListList
```

Comment:
Why not put MeshBufListMaps here?

Reply:
`MeshBufListMaps` (which is just the new name of`MeshBufListList`) is only used in clientmap.cpp, so I thought I'd keep it in the .cpp.
(Initially I just did this so that I don't have to recompile so much. x))
I can move it to header if you request. I don't really care.

Reply:
I don't really care. Maybe I have a slight preference in the header, but do not feel strongly.
What do others think?

Reply:
less stuff in headers = good